rust/src/shards/gui/containers/mod.rs

```diff
@@ -23,14 +23,23 @@ struct Area {
 
 shenum! {
   struct Anchor {
+    [description("Top left corner.")]
```

Comment:
Shouldn't it be like
```rs
#[description("Middle left.")]
const Left = 0x10;
```
to match how attributes look in rust

Reply:
I tried that but then I also can't allow additional attributes at the same time with the ` $(#[$attr:ident $($args:tt)*])*` rule. The macro system thinks it is ambiguous to have:
```rust
$(
  #[description($desc:literal)]
  $(#[$inner:ident $($args:tt)*])*
  const $EnumValue:ident = $value:expr;
)+
```

Now it's possible we will never need to add other attributes but I preferred to be future-proof. In addition it clearly shows that those are not the usual attributes.

Reply:
Yeah okay, makes sense